ci/prow/make_config.go

```diff
@@ -1219,8 +1022,8 @@ func addProjAndRepoIfNeed(projName string, repoName string) map[string][]string
 	return jobDetailMap
 }
 
-// addTestCoverageJobIfNeeded adds test-coverage job for the repo if it has go coverage check
-func addTestCoverageJobIfNeeded(jobDetailMap *map[string][]string, repoName string) {
+// addTestCoverageJobDataIfNeeded adds test-coverage job data for the repo if it has go coverage check
```

Comment:
Since you decided to rename these methods, I think this one should be called "updateTestCoverageJobDataIfNeeded", since it doesn't add anything to the config. Likewise, the function below should keep its original name, since it does add stuff to the config instead of just dealing with data.

Reply:
Changed as suggested.